docs/production/scim.md

```diff
@@ -1,34 +1,65 @@
-# SCIM
+# SCIM provisioning
 
-Zulip has preliminary support for user provisioning and deprovisioning
-via the SCIM protocol, with Okta being the only currently supported provider.
-Support for other providers is expected to be added in the future.
+Zulip has beta support for user provisioning and deprovisioning via
+the SCIM protocol. In SCIM, a third-party SCIM Identity Provider (IdP)
+acts the SCIM client, connecting to the service provider (your Zulip
+server).
```

Comment:
I found the terminology to be likely confusing for someone who isn't super familiar with SCIM, so added these definitions and tried to use terms like "Zulip server" (not just "server") and "SCIM IdP" rather than "the client" to see if that helps. Feel free to tweak further.